media/js/firefox/new/wait-face-scene1-video.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var videoPoster = new Mozilla.VideoPosterHelper('.faster-video');
+
+    function trackVideoInteraction(title, state) {
+        window.dataLayer.push({
+            'event': 'video-interaction',
+            'videoTitle': title,
+            'interaction': state
+        });
+    }
+
+    function initVideoInteractionTracking() {
+        var videos = document.querySelectorAll('video');
```

Comment:
Any reason not to give the `<video>` an `id` and skip the `for` loop below?

Reply:
Bah, yes - this was hastened copy/pasta from another page 👍